CRM/Export/BAO/Export.php

```diff
@@ -330,8 +330,7 @@ public static function exportComponents(
       if (empty($returnProperties['id'])) {
         $returnProperties['id'] = 1;
       }
-
-      $processor->setHouseholdMergeReturnProperties(array_diff_key($returnProperties, array_fill_keys(['location_type', 'im_provider'], 1)));
+      $processor->setHouseholdMergeReturnProperties($returnProperties);
```

Comment:
Location type was missing from the merged household export row because of this.  There seems no sensible reason to explicitly exclude these two fields from merged households!